packages/app/src/app.ts

```diff
@@ -139,7 +143,9 @@ export class AppImpl implements App {
                         );
                         break;
                 }
-            }
+            } else if(response.status == 202) {
+                console.log(await response.text());
```

Comment:
I think I'd rather not include a `console.log` here, as this is a library.

In the future we may have some proper `logger` infrastructure, then we add a log